packages/yuudachi/src/functions/cases/createCase.ts

```diff
@@ -27,10 +31,15 @@ export type CreateCase = Omit<
 > & {
 	caseId?: number | undefined | null;
 	actionExpiration?: Date | undefined | null;
-	modId?: Snowflake | undefined | null;
-	modTag?: string | undefined | null;
+	reason?: string | undefined | null;
+	modId?: Snowflake | undefined;
+	modTag?: string | undefined;
+	targetId: Snowflake;
+	targetTag: string;
+	contextMessageId?: Snowflake | undefined | null;
+	refId?: number | undefined | null;
+	reportRef?: number | undefined | null;
 	multi?: boolean | undefined | null;
```

Comment:
What is happening here?

Reply:
I think that when I resolved the merge conflicts it did that, will take a better look on it

Reply:
Fixed in 33743ae